.circleci/config.yml

```diff
@@ -56,10 +56,23 @@ aliases:
       - attach_workspace:
           at: ~/datadog
       - <<: *STEP_EXT_INSTALL
+      - <<: *STEP_COMPOSER_SELF_UPDATE
+      - <<: *STEP_COMPOSER_CACHE_RESTORE
+      - <<: *STEP_COMPOSER_INSTALL
+      - <<: *STEP_COMPOSER_CACHE_SAVE
+      - run:
+          name: testresults dir
+          command: mkdir test-results
       - run:
           name: export .env.circleci
           command: |
             echo "export $(cat .env.circleci | xargs)" >> $BASH_ENV
+      - run:
+          name: Run extension tests
+          command: TEST_PHP_JUNIT=test-results/simle_test.xml REPORT_EXIT_STATUS=1 make -f DDMakefile test_c
```

Comment:
Actually since we do not need to install the Valgrind anymore this step is no longer necessary. Previously I've done it twice to avoid having to install valgrind if the unit tests themselves are failing - which is an arguable optimization in the first place :).

`test_c_mem` will run tests with valgrind so we'll get both: the info about leaks, and test results.

Reply:
Removing it!